libs/wingc/src/fold.rs

```diff
@@ -87,6 +87,10 @@ where
 			module_name: f.fold_symbol(module_name),
 			identifier: identifier.map(|id| f.fold_symbol(id)),
 		},
+		StmtKind::Module { name, statements } => StmtKind::Module {
```

Comment:
So I am assuming "library" will be used to represent libraries (we need a way to distinguish between module/namespace and a library as a reusable artifact).


@yoav-steinberg what do you think?

Reply:
I'm not sure I see any functional difference between libraries and modules, as far as the type system / compiler is concerned. Both represent a collection of API elements (classes, structs, constants, etc.). My understanding is a library is just a module that a user has exported as an artifact for sharing and reuse.

Reply:
A library, for example, has dependencies.

Reply:
I see. Yeah, thinking ahead, I'm expecting we won't want to inject libraries in the AST - it makes more sense to store library/dependency information inside the type system. But we'll see.

Reply:
Totally agree. My comment was about terminology. If we are cool with calling libraries "libraries", then we can use the term "module" to refer to a namespace.